### PR TITLE
upgrade less.js to match version used on servers

### PR DESCRIPTION
**Pull request by czue**
this should help prevent version incompatibilities between dev mode and production environments

### CODE REVIEW

**Review by orangejenny — APPROVED** [review]
thank you!

**Comment by czue** [comment]
(added one commit to fix a css bug that depended on a higher less version)

File: custom/icds_reports/static/icds_reports/mobile/css/map-chart.less [hunk 1]
```diff
@@ -17,7 +17,7 @@
   .float-unset;
   .p-3;
   .rounded-border;
-  height: calc(100vh - 170px);
+  height: ~"calc(100vh - 170px)";
```
Comment (orangejenny) [inline custom/icds_reports/static/icds_reports/mobile/css/map-chart.less]: Does the `calc` on line 46 also need to be updated?

Comment (orangejenny) [inline custom/icds_reports/static/icds_reports/mobile/css/map-chart.less]: Does the `calc` on line 46 also need to be updated?

Comment (czue) [reply custom/icds_reports/static/icds_reports/mobile/css/map-chart.less]: thanks, not sure how I missed that.

https://github.com/dimagi/commcare-hq/pull/26212/commits/67c4abed33b90bdab6599af8c38794ea24b3ce09

**Pull request merged**